<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>210</y>
      <width>461</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>R/W</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QLabel" name="label_dt2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0" rowspan="2" colspan="5">
      <widget class="QPushButton" name="pushButton_read">
       <property name="text">
        <string>READ ALL EPPROM</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEdit_data_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_dt1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_dt3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <widget class="QPushButton" name="pushButton_pos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>WRITE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_data_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="lineEdit_data_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_data_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Data Write</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="5">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>READ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Data Read</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_dt4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_addr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_data_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="lineEdit_rdata0"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_rdata1"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="lineEdit_rdata2"/>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="lineEdit_rdata3"/>
     </item>
     <item row="7" column="4">
      <widget class="QLineEdit" name="lineEdit_rdata4"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>471</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>EEPROM DATAS</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>28</y>
       <width>451</width>
       <height>471</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>0x00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x01</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x02</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x03</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x04</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x05</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x06</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x07</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x08</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x09</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x0A</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x0B</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x0C</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x0D</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x0E</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x0F</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x1A</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x1B</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x1C</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x1D</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x1E</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>0x1F</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>0</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>6</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>7</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>8</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>9</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>A</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>C</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>E</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>F</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>106</width>
       <height>87</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>1 BYTES</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>5 BYTES</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Cmd</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_rdata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>161</width>
       <height>181</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Serial</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_parity">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_connect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DataSize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_datasize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Baudrate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_baudrate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
